packages/components/src/components/Icon/index.tsx

```diff
@@ -1,10 +1,12 @@
+/* eslint-disable global-require */
 import styled, { keyframes } from 'styled-components';
 
 import PropTypes from 'prop-types';
 import React from 'react';
+import ReactSvg from 'react-svg';
 import colors from '../../config/colors';
-import icons from '../../config/icons';
-import { Omit, IconShape } from '../../support/types';
+import { Omit, IconType } from '../../support/types';
```

Comment:
I the newest version of typescript, omit has already native support. So it might be deleted from support/types.